cmake/ctypesgen.cmake

```diff
@@ -31,7 +31,7 @@ if(NOT MSVC)
 endif()
 set(ENV{LC_ALL} C)
 
-set(CTYPESFLAGS "\"${COMPILER}\" -E ${C_FLAGS}")
+set(CTYPESFLAGS "${COMPILER} -E ${C_FLAGS}")
```

Comment:
We fixed it by quoting in https://github.com/OSGeo/grass/pull/6298 and now you're reverting it?